multirun/test/BUILD.bazel

```diff
@@ -133,6 +133,16 @@ multirun(
     parallel = True,
 )
 
+multirun(
```

Comment:
This is not testing anything, would be great to add a test.

Reply:
added a proper test